### PR TITLE
another small fix for jenkins...

### PR DESCRIPTION
**Pull request by gocarlos**

### CODE REVIEW

**Comment by gocarlos** [comment]
I think now should be good to merge it.

**Comment by ffurrer** [comment]
Not sure, but I guess you can then remove `find_package(librealsense QUIET)`. 

Other than that lgtm

File: InfiniTAM/CMakeLists.txt [hunk 1]
```diff
@@ -33,10 +33,6 @@ find_package(libuvc QUIET)
 CANONIFY_BOOL(libuvc_FOUND)
 message(STATUS "libuvc found: ${libuvc_FOUND}")
 
-find_package(librealsense QUIET)
-CANONIFY_BOOL(librealsense_FOUND)
```
Comment (ffurrer) [inline InfiniTAM/CMakeLists.txt:37]: These lower two lines are still required for the rest of infinitam to compile with librealsense, I only talked about the first line (find_package) that you can omit.

Comment (gocarlos) [reply InfiniTAM/CMakeLists.txt:37]: Alright :)

**Pull request merged**